hrjobcontract/scss/civihr/_bootstrap-variables.sass

```diff
@@ -175,11 +176,17 @@ $btn-default-border:             $gray-light
 // $btn-warning-border:             darken($btn-warning-bg, 5%)
 
 // $btn-danger-color:               #fff
-$btn-danger-bg:                  #e6807f
+$btn-danger-bg:                  #CF3458
 $btn-danger-border:              $btn-danger-bg
 
 // $btn-link-disabled-color:        $gray-light
 
+$brand-primary-dark:             #005A97
+$brand-secondary-dark:           #3D3D54
+$brand-success-dark:             #2C8452
+$brand-info-dark:                #005A97
+$brand-warning-dark:             #9F6A26
+$brand-danger-dark:              #A52946
```

Comment:
why do we need so many new colors? We should replace the existing colors, not add new ones

Reply:
We have introduced this colors in new color palette, for hover and active states of text/buttons.
Ref. https://invis.io/7QCECB68Z#/254761833_WCAG_AA_Options

Reply:
but in shoreditch we didn't have to manually define them, if i recall correctly the original colors are darkened/lightened via SASS automatically. Why do we need to do it here?

Reply:
In new palette color, we have done colors like this in shoreditch as well as the darken variation is from 7% to 25% for dark light buttons,
i.e. primary `#005A97` is 7% darker of `#0071BD` while warning `#9F6A26` is 25% darker of `#E6AB5E`

Reply:
then this is an inconsistency of the color palette, please talk with jon about it. Ideally the percentage should be the same as in the original bootstrap (so that the hover/active/disabled colors are applied automatically without any need of intervention from us), but at the very least the variation should be consistent across all colors

We really can't have colors explicitly defined for hover/active/disabled variations, it muddies the color palette which should be as simple and concise as possible

Reply:
@jonscreat, ^ can you please see this? 
@AkA84, I will discuss with Jon for this